src/ripple/app/misc/TxQ.h

```diff
@@ -319,6 +319,25 @@ class TxQ
     Metrics
     getMetrics(OpenView const& view) const;
 
+    /**
+     * @brief Returns minimum required fee for tx and two sequences:
+     *        first vaild sequence for this account in current ledger
+     *        and first available sequence for transaction
+     * @param view current open ledger
+     * @param tx the transaction
+     * @param accountSeq reference to save first sequence in the ledger
+     * @param availableSeq reference to save firts available sequence
+     * @param useFeeIncrease increase minimum required fee in the case
+     *        of multi tx and in the case of replace existing tx
+     * @return minimum required fee
+     */
+    XRPAmount
+    getTxRequiredFeeAndSeq(OpenView const& view,
+        std::shared_ptr<STTx const> const& tx,
+        std::uint32_t &accountSeq,
+        std::uint32_t &availableSeq,
+        bool useFeeIncrease = false) const;
```

Comment:
I'd get rid of the return value combined with the out parameters and just return a struct with the values you want to return. (Also, style nit: we usually write `uint32_t&` instead of `uint32_t &`)

Reply:
Done.